Update platform-views.md (#8799)
flutter/website: src/platform-integration/android/platform-views.md

   <?code-excerpt "lib/platform_views/native_view_example_1.dart (HybridCompositionWidget)"?>
   ```dart
   Widget build(BuildContext context) {

>      // 'viewType' here, is used in the platform side to register the view.
>      // It works like an ID, it can be any safe string

Reviewer: **issue:** Write in active voice per [GSG](https://developers.google.com/style/voice).
```suggestion
     // This 'viewType' string registers the view on the platform side.
     // It works like an ID, it can be any safe string.
```
